### PR TITLE
Resolves #2108

### LINKED ISSUE #2108: Error in LogLevel Parameter description of Add-WindowsCapability
[Enter feedback here]

The LogLevel section in the Add-Windowscapability documentation first states 4 acceptable numeric values and their meaning while the later "Accepted values" subsection only states 3 enum values (Errors, Warnings, WarningsInfo). By my testing the latter seems to be correct, Numbers do not work and there are only 3 possible settings for the LogLevel switch (which is a little unfortunate because a verbose/debug one would be nice)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 4ce94c11-8f4f-89b3-1d87-b349bfb63889
* Version Independent ID: b175176e-2c8c-8869-2e43-6c45f4ed5671
* Content: [Add-WindowsCapability (dism)](https://docs.microsoft.com/en-us/powershell/module/dism/add-windowscapability?view=win10-ps)
* Content Source: [docset/windows/dism/Add-WindowsCapability.md](https://github.com/MicrosoftDocs/windows-powershell-docs/blob/master/docset/windows/dism/Add-WindowsCapability.md)
* Product: **w10**
* Technology: **powershell-windows**
* GitHub Login: @andreabarr
* Microsoft Alias: **v-anbarr**

### PR DESCRIPTION
**Pull request by velkovb**
Fix parameter description
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/2108

### CODE REVIEW

**Comment by e0i** [comment]
@dariomws @get-itips @tseward

Peer review needed for this PR.

cc @JohanFreelancer9 for copy editing.

Thank you.

**Review by get-itips — APPROVED** [review]

File: docset/windows/dism/Add-WindowsCapability.md [hunk 1]
```diff
@@ -84,12 +84,6 @@ Accept wildcard characters: False
 
 ### -LogLevel
 Specifies the maximum output level shown in the logs.
```
Comment (dariomws) [inline docset/windows/dism/Add-WindowsCapability.md:86]:
```suggestion
Specifies the maximum output level shown in the logs. The accepted values are as follows:
- Errors
- Warnings
- WarningsInfo (default)
```

**Review by dariomws — CHANGES_REQUESTED** [review]

File: docset/windows/dism/Add-WindowsCapability.md (hunk 1 above)
Comment (velkovb) [reply docset/windows/dism/Add-WindowsCapability.md:86]: I removed it because it is actually included in the` Accepted Values: ` section.

**Review by dariomws — APPROVED** [review]
@velkovb agree we can leave them just there. Thanks!

**Review by tseward — APPROVED** [review]

**Pull request merged**